### PR TITLE
VACMS-17178: Update CLP Links to V3 components

### PR DESCRIPTION
**Pull request by chriskim2311**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
Update CLP links to use V3 components

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17178

## Testing done
Tested locally

## Screenshots
Sign_In_Securely_With_Login_gov___Veterans_Affairs

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### CODE REVIEW

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 1]
```diff
@@ -315,8 +311,8 @@
                   href="{{ resource.entity.fieldMediaExternalFile.uri }}"
                   onclick="recordEvent({ 'event': 'clp-link-click', 'clp-section-category': 'Downloadable resources', 'clp-section-title': '{{ fieldClpResourcesHeader | encode }}', 'clp-total-sections': '{{ clpTotalSections }}', 'clp-click-label': 'Download {{ resource.entity.name | encode }}' });"
                 >
-                  Download (PDF)
-                </a>
+                Download (PDF)
```
[the remark this answers is not in the document]
Comment (chriskim2311) [reply src/site/layouts/campaign_landing_page.drupal.liquid:314]: Thanks, re-read your comment about pdf downloads and misunderstood thought it was for all pdf downloads. I will add the va-link here.

[the remark this answers is not in the document]
Comment (randimays) [reply src/site/layouts/campaign_landing_page.drupal.liquid:314]: Oh, thank you for reminding me about that! You're right - you interpreted the comment about PDF downloads correctly. This should **not** be using a `<va-link>`. Apologies for the confusion!

[the remark this answers is not in the document]
Comment (chriskim2311) [reply src/site/layouts/campaign_landing_page.drupal.liquid:314]: Got it thanks for the sanity check here!

**Comment by chriskim2311** [comment]
@randimays I addressed the missing pieces from your comments. The accordion bug has been ticketed with DST and they will resolve the issue.

File: src/site/layouts/campaign_landing_page.drupal.liquid [hunk 2]
```diff
@@ -653,13 +653,14 @@
                     class="vads-u-flex--auto icon-small white vads-u-margin-right--1 hub-icon-{{ benefitCategory.entity.fieldTitleIcon }} hub-background-{{ benefitCategory.entity.fieldTitleIcon }}"
                     role="presentation"
                   ></i>
-                  <a
+                  <va-link
```
Comment (randimays) [inline src/site/layouts/campaign_landing_page.drupal.liquid:656]: This needs to be wrapped in the `h3` for semantics and style, please! Ours vs prod:

Screenshot 2024-03-12 at 3 49 34 PM
Screenshot 2024-03-12 at 3 49 30 PM

**Review by randimays — CHANGES_REQUESTED** [review]

**Comment by laflannery** [comment]
@chriskim2311 I have 1 update and 2 questions:
- Actual update: You can remove `role="presentation"` from the icons (social links at the top and Benefits at the bottom) because you already have `aria-hidden`.
- The social links open in a new window, how did you do this? I can't see anything in the code that is making this happen (e.g. target="_blank") and we were pretty sure the link component didn't allow this
- This is the only ticket so far we've updated the Feedback button - I think I was under the impression this was more of a overall sitewide update, not a template by template thing? I do not want an aria-label here (I know that there currently is one), would that be my decision to make? Or is that a larger decision, do we know?

**Comment by jamigibbs** [comment]
> There's also a bug with accordions on CLPs. When you hover over the button, the hovered plus icon gets small and thinner. If possible, could we fix this as part of this PR as well?

@chriskim2311 @randimays Just a head's up that we have recently fixed the issue of the accordion icons resizing on hover. That update will be in the next component-library release:

- https://github.com/department-of-veterans-affairs/component-library/pull/1083

File: src/site/layouts/campaign_landing_page.drupal.liquid (hunk 2 above)
Comment (randimays) [reply src/site/layouts/campaign_landing_page.drupal.liquid:656]: @chriskim2311 I think we lost some styles here controlling the look of the `<h3>`

Screenshot 2024-03-14 at 3 55 47 PM

**Review by randimays — APPROVED** [review]
LGTM!

**Comment by laflannery** [comment]
Looks good, thanks!

**Pull request merged**